scp/lib/main.dart

```diff
@@ -299,13 +314,13 @@ class _HomePageState extends State<HomePage> {
             DateFormat.MMM().format(now.subtract(Duration(days: 1))));
   }
 
-  _startFAQActivity() async {
-    try {
-      await platform.invokeMethod('startFaqActivity');
-    } on PlatformException catch (e) {
-      print(e.message);
-    }
-  }
+  // _startFAQActivity() async {
+  //   try {
+  //     await platform.invokeMethod('startFaqActivity');
+  //   } on PlatformException catch (e) {
+  //     print(e.message);
+  //   }
+  // }
```

Comment:
This comment is intentional, right?

Reply:
Yes, it is. The function was unused. FAQ activity works correctly, even without the method. The purpose of that function is still unknown, so it is commented out.